hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/mover/MoverStatus.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ssm.mover;
+
+import org.apache.hadoop.util.Time;
+
+/**
+ * Status of Mover tool.
+ */
+public class MoverStatus extends Status {
+  private Boolean isFinished = false;
+  private long startTime = 0;
+  private Boolean succeeded = false;
+  private long totalDuration = 0;
+
```

Comment:
it's good enough for starting, but we'd better to provide more insightful metrics later.